<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="page_3"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionBookManage"/>
   <addaction name="separator"/>
   <addaction name="actionReaderManage"/>
   <addaction name="separator"/>
   <addaction name="actionBorrowManage"/>
   <addaction name="separator"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionBookManage">
   <property name="text">
    <string>图书管理</string>
   </property>
  </action>
  <action name="actionReaderManage">
   <property name="text">
    <string>读者管理</string>
   </property>
  </action>
  <action name="actionBorrowManage">
   <property name="text">
    <string>借阅管理</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
